packages/node/src/login/oidc/ClientRegistrar.ts

```diff
@@ -143,18 +149,30 @@ export default class ClientRegistrar implements IClientRegistrar {
       }
     );
 
+    let clientExpiresAt = 0;
+    if (
+      typeof registeredClient.metadata.client_secret_expires_at === "number"
```

Comment:
Note: `openid-client` doesn't actually verify that `client_secret_expires_at` is present and that it is in the future, I've added a set of checks in oidc-client-ext to verify these for the browser, but for server we don't currently have these checks.

Reply:
In theory here we could receive back a client with a `client_secret_expires_at` in the past, if the users' clock time on their server is ahead of the IdP's clock time.

Can we assume that users and IdP's can configure their servers to have a synchronised clock time?